# Automatically Generated in 2017-06-12 22:33:47.452753
# MESSAGE: TELEMETRY
# Description:Telemetry
uint8 ID = 3
uint8 sysid
uint8 compid
uint32 timestamp
float32 xacc
float32 yacc
float32 zacc
float32 xgyro
float32 ygyro
float32 zgyro
float32 dist_front
float32 dist_rear
float32 dist_side
float32 odom
float32 odom_accumulated
float32 xmotion_front
float32 ymotion_front
float32 xmotion_rear
float32 ymotion_rear
uint8 motion_front_quality
uint8 motion_rear_quality
int32 current_motor
uint16 current_servo_front
uint16 current_servo_rear
uint16 current_total
uint16 pwm_servo_front
uint16 pwm_servo_rear
uint16 battery_voltage
uint8 remote_control
uint8 drive_mode
